jdbc/src/main/java/org/jclouds/jdbc/entity/ContainerEntity.java

```diff
@@ -96,6 +96,30 @@ public static Builder builder() {
       return new Builder();
    }
 
+   @Override
+   public boolean equals(Object o) {
+      if (this == o)
+         return true;
+      if (o == null || getClass() != o.getClass())
+         return false;
+
+      ContainerEntity containerEntity = (ContainerEntity) o;
+
+      return containerAccess == containerEntity.containerAccess && !(creationDate != null ?
+            !creationDate.equals(containerEntity.creationDate) :
+            containerEntity.creationDate != null) && !(id != null ? !id.equals(containerEntity.id) : containerEntity.id != null) && !(
+            name != null ? !name.equals(containerEntity.name) : containerEntity.name != null);
+   }
+
+   @Override
+   public int hashCode() {
```

Comment:
Use guava helpers for hashcode/equals/tostring, to reduce code:
https://code.google.com/p/guava-libraries/wiki/CommonObjectUtilitiesExplained